crates/bevy_render/src/view/mod.rs

```diff
@@ -74,15 +75,36 @@ pub struct Msaa {
     /// Note that WGPU currently only supports 1 or 4 samples.
     /// Ultimately we plan on supporting whatever is natively supported on a given device.
     /// Check out this issue for more info: <https://github.com/gfx-rs/wgpu/issues/1832>
-    pub samples: u32,
+    pub level: MultiSampleLevel,
+}
+
+impl Msaa {
+    pub fn samples(&self) -> u32 {
```

Comment:
```suggestion
    #[inline]
    pub fn samples(&self) -> u32 {
```

this should probably be inlined